### ISSUE TITLE
Update PDFs to latest version

### ISSUE THREAD

**Issue by aviaryan** [aviaryan/learnxinyminutes-pdf#6]

**Comment by aashutoshrathi** [comment]
Can't we automate this? Using CI and GH Token.

**Comment by aviaryan** [comment]
@aashutoshrathi Yes, it is possible. There are [some static configs](https://github.com/aviaryan/learnxinyminutes-pdf/blob/master/genpdf.py#L12-L32) which need to be checked from time to time but apart from that, running the scripts as mentioned in the README does the trick.

**Comment by nbehrnd** [comment]
There are two submitted revisions about `genpdf.py` ([here](https://github.com/aviaryan/learnxinyminutes-pdf/pull/8)) and `_genpdf.sh` ([here](https://github.com/aviaryan/learnxinyminutes-pdf/pull/9)) to account for pandoc's different keyword to generate .pdf.  Given the latest release of this project is by August 2016, and the ongoing growth of the repository learnxinyminutes-doc (e.g., now including AWK), I equally support the suggestion to prepare a new release of this handy reference.
